src/runtime/pyint.cs

```diff
@@ -51,7 +51,10 @@ public PyInt(PyObject o)
         public PyInt(int value)
         {
             obj = Runtime.PyInt_FromInt32(value);
-            Runtime.CheckExceptionOccurred();
+            if (obj == IntPtr.Zero)
+            {
+                throw new PythonException();
+            }
```

Comment:
Why? `Runtime.CheckExceptionOccurred()` is shorter and equivalent, according to the Python documentation.